docs/alphafeatures.md

```diff
@@ -423,7 +423,7 @@ The following section will give the steps to take snapshot and clone the cStor v
    ```
 3. Get the YAML for snapshot creation of a PVC using the following command:
    ```
-   wget https://raw.githubusercontent.com/openebs/cstor-csi/master/deploy/snapshot.yaml
+   wget https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-snapshot.yaml
    ```
    In this example, downloaded file is saved as `snapshot.yaml`.
```

Comment:
In this example, downloaded file is saved as `snapshot.yaml`. -> In this example, downloaded file is saved as `csi-snapshot.yaml`.